main.tf

```
@@ -0,0 +1,20 @@
+terraform {
+  cloud {
+    organization = "liamstevens111"
+
+    workspaces {
+      name = "liam-infrastructure-example"
+    }
+  }
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = "us-east-1"
```

Why `US East (N. Virginia)`, why not use [Singapore region](https://aws.amazon.com/financial-services/security-compliance/compliance-center/sg/) to have a better latency 🤔 

```
ap-southeast-1
```

---

because we had resources already on Nimble account so I would separate the region (and ap-southeast lacks some services) but I ended up putting in my own account anyway

---

So you will you your account? AWS account? will we use Singapore region or US East (N. Virginia)? Anw you can decide it hehe, not a big deal

---

Should we move this region value into a variable? So we can change it later via TF Cloud.
And we can manage the region by using that variable instead of the hardcode:

```suggestion
  region = var.region
```

And in the `variables.tf`:
```
variable "region" {
  description = "AWS region"
  type        = string
  default     = "us-east-1"
}
```